src/routes/docs/tooling/keyboard/+page.markdoc

```diff
@@ -0,0 +1,39 @@
+---
+layout: article
+title: Keyboard
+description: How to use the keyboard with the optimized Appwrite Console for both efficiency and accessibility, and introducing The Appwriter.
+---
+
+The Appwrite SDK console is optimized with the keyboard using shortcut keys to enable accessibility.  Using shortcut keys access websites without the mouse. 
```

Comment:
I would consider including some more context to the reader, why its great to have keyboard optimization and accessibility, ease of use, fast way to perform important actions, great for productivity, enabling usage for people with disabilities who rely solely on the keyboard.